pytrip/utils/cubeslice.py

```diff
@@ -200,6 +200,10 @@ def main(args=sys.argv[1:]):
     ct_im = None
     data_im = None
 
+    if args.sstart == 0:
+        logger.error("First slice must be 1 or greater.")
```

Comment:
I'd recommend also to provide an information about value provided by user, i.e.:
"you provided XXXX, but first slice must be 1 or greater"